NitroxModel/DataStructures/GameLogic/Entity.cs

```diff
@@ -1,74 +1,31 @@
-﻿using System;
+using System;
 using System.Collections.Generic;
 using BinaryPack.Attributes;
+using NitroxModel.DataStructures.GameLogic.Entities;
 using NitroxModel.DataStructures.GameLogic.Entities.Metadata;
-using NitroxModel.DataStructures.Unity;
 using ProtoBufNet;
 
 namespace NitroxModel.DataStructures.GameLogic
 {
     [Serializable]
     [ProtoContract]
-    public class Entity
+    [ProtoInclude(50, typeof(WorldEntity))]
+    [ProtoInclude(60, typeof(PrefabChildEntity))]
+    public abstract class Entity
     {
-        public AbsoluteEntityCell AbsoluteEntityCell => new AbsoluteEntityCell(Transform.Position, Level);
-
         [ProtoMember(1)]
-        public NitroxTransform Transform { get; set; }
+        public NitroxId Id { get; set; }
```

Comment:
With the protomember changes if the ids are kept the same on the properties and newer ones are using the newer ids, upgrading with protobuf should then continue as protobuf is about to handle property changes as long as the IDs don't change for that property.

I do appreciate this does give weird numberings. I guess it depends on our preference for backwards compatibility

Reply:
Protobuf will break anyway bc the Entity system will not be reusable with old saves with my changes